src/Mutator/Removal/FunctionCallRemoval.php

```diff
@@ -11,6 +11,7 @@
 
 use Infection\Mutator\Util\Mutator;
 use PhpParser\Node;
+use function strtolower;
```

Comment:
We don't do this anywhere within the code base, so please revert this change.

Reply:
on the other side - we can discuss it and start using this approach, at least understand core developers think about it.

Personally, I find it more readable as it removes all those `\`,

`\assert(\count($a) > 0)` vs `assert(count($a) > 0)`

@infection/core